lib/activity_model.dart

```diff
@@ -205,6 +208,50 @@ class ActivityModel extends Model {
     unMaskImagePath = text;
     notifyListeners();
   }
+
+  Color color = Colors.white;
+  BlendMode blendMode = BlendMode.dst;
+  String imageId;
+
+  void filterImage(String text) {
+    Color cls;
+    BlendMode blnd;
+    if (text == 'assets/filter_image/default.png') {
```

Comment:
The model should not be tied to the button name. You can pass in a enum for each type